### PR TITLE
feat(plugins): add minimap plugin

### PR DESCRIPTION
**Pull request by Aarebecca**
Jul-31-2024 22-58-22

- [x] 新增缩略图插件
- [x] 完善相关测试，基于 playwright 进行截图对比

> 目前没有合并 jest 和 playwright 的总体覆盖率，而是先将通过 playwright 进行测试的部分从 jest 覆盖率中排除
---

- [x] Added minimap plugin 
- [x] Improved related tests, screenshots were compared based on playwright

> > Overall coverage of jest and playwright is not currently combined, but portions tested through playwright are excluded from jest coverage first

### CODE REVIEW

File: packages/g6/src/plugins/minimap/index.ts [hunk 1]
```diff
@@ -0,0 +1,514 @@
...
+   * - 'key' 使用元素的主图形作为缩略图形
+   * - 'symbol' 使用固定的图形表示元素
+   * - 也可以传入一个函数，接收元素的 id 和类型，返回一个图形
+   *
+   * <en/>
+   * - 'key' uses the key shape of the element as the thumbnail graphic
+   * - 'symbol' uses a fixed shape to represent the element
+   * - You can also pass in a function that receives the id and type of the element and returns a shape
+   */
+  shape?: 'key' | ((id: string, elementType: ElementType) => DisplayObject);
```
Comment (yvonneyx) [inline packages/g6/src/plugins/minimap/index.ts:66]: 这里漏写了 symbol 嗷

**Review by yvonneyx — APPROVED** [review]

File: packages/g6/src/plugins/minimap/index.ts (hunk 1 above)
Comment (Aarebecca) [reply packages/g6/src/plugins/minimap/index.ts:64]: @yvonneyx 这里最初设想可以传入 `key` 、字符串描述的 symbol，以及 DisplayObject，不过后来看第一种和第三种已经足够灵活了。现在把 symbol 相关注释移除了。

**Comment by hustcc** [comment]
- fixed https://github.com/antvis/G6/issues/5560

**Comment by nzjcnjzx** [comment]
当前插件，不展示自定义节点的文案和图片

**Pull request merged**